templatetags/field_updater_tags.py

```diff
@@ -34,6 +36,8 @@ def field_updater(
             'submit_url': submit_url,
             'bodyEncode': body_encode,
             'prefix': prefix,
+            'headersAccept': headers_accept,
+            'headersContentType': headers_content_type,
```

Comment:
content type header is already automatically set by fetch!

Reply:
Ah, yes! I always forget that. 🤦